ocs_ci/framework/main.py

```diff
@@ -45,14 +46,20 @@ def init_ocsci_conf(arguments=None):
         with open(os.path.expanduser(cluster_config)) as file_stream:
             cluster_config_data = yaml.safe_load(file_stream)
             framework.config.update(cluster_config_data)
+    framework.config.RUN['run_id'] = int(time.time())
 
 
 def main(arguments):
     init_ocsci_conf(arguments)
+    pytest_logs_dir = os.path.join(os.path.expanduser(
+        framework.config.RUN['log_dir']
+    ), f"pytest-logs-{framework.config.RUN['run_id']}")
```

Comment:
s/pytest-logs/ocs-ci/

Reply:
@vasukulkarni the default location used by pytest is `/tmp/pytest-of-<username>/pytest-X/`, that was the reason, why I put the `pytest` prefix there.
If you think, it should rather contain `oc-ci` prefix, I would propose to use `ocs-ci-logs`, WDYT?